tests/smoke-tests/test-ethereum-db-mapping.ts

```diff
@@ -0,0 +1,40 @@
+import { expect } from "chai";
+import { describeSmokeSuite } from "../util/setup-smoke-tests";
+
+const debug = require("debug")("smoke:ethereum-mapping");
+
+const wssUrl = process.env.WSS_URL || null;
+const ethRpcUrl = process.env.HTTPS_URL || null;
+const relayWssUrl = process.env.RELAY_WSS_URL || null;
+
+// At rpc-level, `*ByNumber` requests always use the canonical block reference given by Substrate.
+// In the other hand `*ByHash` requests rely on data mapped in the frontier db.
+// We want to compare both to verify recent db data consistency and rpc impl across client versions.
+
+describeSmokeSuite(
+  `Ethereum secondary DB should contains valid data`,
+  { wssUrl, relayWssUrl, ethRpcUrl },
+  (context) => {
+    it("should get the same response payload on byNumber and byHash requests", async function () {
+      this.timeout(6_000_000); // 30 minutes
```

Comment:
That seems long, we should always try to target reasonable time, with a bit of margin. 
Here it seems to query 6 blocks with 2 requests each. It shouldn't be more than 20s